chips/msp432/src/usci.rs

```diff
@@ -102,6 +102,147 @@ register_structs! {
 }
 
 register_bitfields![u16,
+    pub(crate) UCSPIxCTLW0 [
+        /// Software reset enable
+        UCSWRST OFFSET(0) NUMBITS(1) [
+            /// Disabled. eUSCI reset released for operation
+            Disabled = 0,
+            /// Enabled. eUSCI logic held in reset state
+            Enabled = 1
+        ],
+        /// STE mode select in master mode
+        UCSTEM OFFSET(1) NUMBITS(1) [
+            /// STE pin is used to prevent conflicts with other masters
+            PreventConflicts = 0,
+            /// STE pin is used to generate the enable signal for a 4-wire slave
+            ChipSelect = 1
+        ],
```

Comment:
I am trying to verify the safety of the transmute used in `spi.rs`. I am confused, however, because this bitfield is clearly different than the bit described at offset 1 in `UCBxCTLW0`. I suspect it is because I do not understand how these registers work on the MSP432. Is the idea that these registers have a totally different meaning depending on if the module is configured for SPI vs. if it is configured for I2C? And thus the bitfield descriptions are different in that case?